### ISSUE TITLE
Ep1: Contradiction in working directory guidelines?

### ISSUE THREAD

**Issue by gcapes** [datacarpentry/python-ecology-lesson#357]
This section appears to suggest keeping everything in one folder, before going on to recommend organising into subfolders. The wording probably just needs tweaking but I don't think it makes sense as it stands.

https://datacarpentry.org/python-ecology-lesson/00-before-we-start/index.html#research-project-best-practices

**Comment by wrightaprilm** [comment]
Right, we've been in a bit of a holding pattern with this. We're meant to be moving to a solution where data are stored on FigShare, but we still have one file (bouldercreek) that isn't on the FigShare. Once @fmichonneau has that added in, we'll do an audit to make sure the data instructions are clear.

**Comment by btovar** [comment]
@gcapes Thanks for your report! It seems that this was fixed but the issues was not closed.
